### PR TITLE
Docs: Updated changelog for 6.1 release

### PR DESCRIPTION
**Pull request by torkelo**
Also includes an update to changelog cli task to group changelog into bug fixes and non bug fixes. 

Bug fixes are identified with PR titles that contain the word Fix or Fixes (case unimportant), or the label type/bug

### CODE REVIEW

**Review by dprokop — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,24 +1,57 @@
...
+* **Heatmap**: You can now choose to hide buckets with zero value. [#15934](https://github.com/grafana/grafana/pull/15934), [@alexanderzobnin](https://github.com/alexanderzobnin)
 * **Heatmap**: `Middle` bucket bound option [#15683](https://github.com/grafana/grafana/issues/15683)
 * **Heatmap**: `Reverse order` option for changing order of buckets [#15683](https://github.com/grafana/grafana/issues/15683)
+* **Prometheus**: Change alignment of range queries to end before now and not in future. [#16110](https://github.com/grafana/grafana/pull/16110), [@davkal](https://github.com/davkal)
+* **Prometheus**: Dedup annotations events with same timestamp . [#16152](https://github.com/grafana/grafana/pull/16152), [@torkelo](https://github.com/torkelo)
+* **SQL**: Use default min interval of 1m for all SQL data sources. [#15799](https://github.com/grafana/grafana/pull/15799), [@marefr](https://github.com/marefr)
+* **Support list and revoke of user auth tokens in HTTP API**. [#15839](https://github.com/grafana/grafana/pull/15839), [@marefr](https://github.com/marefr)
```
Comment (marefr) [inline CHANGELOG.md]: This is a duplicate of the first minor PR.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,24 +1,57 @@
...
+* **Heatmap**: Negative values are now displayed correctly in graph & legend. [#15953](https://github.com/grafana/grafana/pull/15953), [@alexanderzobnin](https://github.com/alexanderzobnin)
 * **Heatmap**: legend shows wrong colors for small values [#14019](https://github.com/grafana/grafana/issues/14019)
+* **InfluxDB**: Always close request body even for error status codes. [#16207](https://github.com/grafana/grafana/pull/16207), [@ramongtx](https://github.com/ramongtx)
+* **ManageDashboards**: Fix for checkboxes not appearing properly Firefox . [#15981](https://github.com/grafana/grafana/pull/15981), [@srid12](https://github.com/srid12)
+* **Playlist**:  Leaving playlist now always stops playlist . [#15791](https://github.com/grafana/grafana/pull/15791), [@peterholmberg](https://github.com/peterholmberg)
+* **Table**: Fix for white text on white background when value is null. [#16199](https://github.com/grafana/grafana/pull/16199), [@peterholmberg](https://github.com/peterholmberg)
+* **Gauge**: Interpolate scoped variables in repeated gauges [#15739](https://github.com/grafana/grafana/issues/15739)
```
Comment (marefr) [inline CHANGELOG.md]: Bug fixes don't seems to be sorted by area?

**Review by marefr — CHANGES_REQUESTED** [review]
Found some minor things

File: CHANGELOG.md (hunk 2 above)
Comment (torkelo) [reply CHANGELOG.md]: did manual merge of the existing items in the changelog forgot to move this to the correct spot

**Review by marefr — APPROVED** [review]

**Pull request merged**